x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/SearchableSnapshotActionIT.java

```diff
@@ -67,6 +67,7 @@ public void refreshIndex() {
         dataStream = "logs-" + randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
         policy = "policy-" + randomAlphaOfLength(5);
         snapshotRepo = randomAlphaOfLengthBetween(4, 10);
+        logger.info(" before [{}]: ds [{}], policy [{}], snapshotRepo [{}]", getTestName(), dataStream, policy, snapshotRepo);
```

Comment:
I think it would be nice if we prefixed the test logging with `-->` like we do for test logging in most other places. Makes it easier to analyze the timing of things in practice :) I'm not sure how much this really helps us either thoguh, we should get the names of these things from the cluster logs anyway?